### PR TITLE
Upgrade to account for changes to @site.pages and @site.documents

### PR DESCRIPTION
**Pull request by de-fluistervloot**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [x] Yes
- [ ] No

There isn't any effort to support 0.x versions of bridgetown

**The PR fulfills these requirements:**

- [x] All tests are passing
- [x] New/updated tests are included

### CODE REVIEW

**Comment by andrewmcodes** [comment]
thanks @de-fluistervloot - I am in the process of contacting Ruby gems to see if I can become the owner of this gem. 

I want to test this myself but my only concern is that this may not be backwards compatible. If it isn’t, I’ll have to think about releasing a major version vs. adding version checks and appraisal tests. Feedback welcome.

**Comment by andrewmcodes** [comment]
@de-fluistervloot I have enabled CI for this branch and looks like the tests are failing. Can you address?

**Comment by de-fluistervloot** [comment]
> thanks @de-fluistervloot - I am in the process of contacting Ruby gems to see if I can become the owner of this gem.
> 
> I want to test this myself but my only concern is that this may not be backwards compatible. If it isn’t, I’ll have to think about releasing a major version vs. adding version checks and appraisal tests. Feedback welcome.

Shouldn't be too hard to support both I guess. But I'm not familiar enough with setting up testing against both 0.x and 1.x versions.

**Comment by andrewmcodes** [comment]
Giving this a test today.

File: .github/workflows/rspec.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 name: RSpec
 
+env:
+  BRIDGETOWN_VERSION: 1.0.0.beta1
```
Comment (andrewmcodes) [inline .github/workflows/rspec.yml:5]: smart!

**Review by andrewmcodes — APPROVED** [review]
Bridgetown v1 does not support Ruby 2.6 so that's why that is failing. This looks good though, thanks for the PR!

**Pull request merged**